spec/models/review_spec.rb

```diff
@@ -25,4 +25,13 @@
     )
     expect(subject).to_not be_valid
   end
+
+  it "raises PG database error without text" do
+    subject = described_class.new(
+      item: @bacon,
+      text: nil
+    )
+
+    expect {subject.save!(validate: false) }.to raise_error(ActiveRecord::NotNullViolation, /PG::NotNullViolation/)
```

Comment:
Looking at this a day later, I want to specifically callout using regex to match /PG::NotNullViolation/ : How dependent should testing be on the fact that we are using postgres, and thus postgres error messages, as our database?

Reply:
I see two options:

1. You could just remove the second argument passed to `raise_error`. In this case, the class name seems sufficient.
2. I think there are built in matchers for `rspec-rails` that you could leverage.